tests/utils/pod/pod.go

```diff
@@ -72,3 +75,20 @@ func RedefineWithPVC(pod *corev1.Pod, name string, claim string) *corev1.Pod {
 
 	return pod
 }
+
+func RedefineWithResources(pod *corev1.Pod, limit string, req string) {
```

Comment:
Same here. "RedefineWithCpuResources" is more accurate.

Reply:
Fixed